> Hi!
> 
> Can you please add a unit test covering this change?

Done!

Passes locally:

```
(venv) carles@pinux:[fix-error-multithread]~/git/django-compressor$ django-admin test --settings=compressor.test_settings compressor -k OfflineCompressPerNodeLockingTestCase
Found 2 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
..
----------------------------------------------------------------------
Ran 2 tests in 3.004s

OK
Destroying test database for alias 'default'...
```

Reverting the commit with the fix, it fails:

```
(venv) carles@pinux:[fix-error-multithread]~/git/django-compressor$ django-admin test --settings=compressor.test_settings compressor -k OfflineCompressPerNodeLockingTestCase
Found 2 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
.F
======================================================================
FAIL: test_same_node_is_not_rendered_concurrently (compressor.tests.test_offline.OfflineCompressPerNodeLockingTestCase.test_same_node_is_not_rendered_concurrently)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/carles/git/django-compressor/compressor/tests/test_offline.py", line 1037, in test_same_node_is_not_rendered_concurrently
    self._assert_concurrency(nodes, nodes, expected_max_active=1)
  File "/home/carles/git/django-compressor/compressor/tests/test_offline.py", line 1028, in _assert_concurrency
    self.assertEqual(max_active[0], expected_max_active)
AssertionError: 2 != 1

----------------------------------------------------------------------
Ran 2 tests in 2.007s

FAILED (failures=1)
Destroying test database for alias 'default'...
(venv) carles@pinux:[fix-error-multithread]~/git/django-compressor$ 
```

--- a/compressor/tests/test_offline.py
+++ b/compressor/tests/test_offline.py
@@ -3,13 +3,14 @@ import io
 import os
 from contextlib import contextmanager
 from importlib import import_module
+import threading
 from unittest import SkipTest
 from unittest.mock import patch
+
 
 from django.conf import settings
 from django.core.management import call_command, CommandError
 from django.template import Context, Origin, Template
-from django.test import override_settings, TestCase
 from django.urls import get_script_prefix, set_script_prefix
 
 from compressor.cache import flush_offline_manifest, get_offline_manifest
@@ -17,6 +18,12 @@ from compressor.exceptions import OfflineGenerationError
 from compressor.management.commands.compress import Command as CompressCommand
 from compressor.storage import default_offline_manifest_storage
 from compressor.utils import get_mod_func
+
+
+import time
+import types
+
+from django.test import TestCase, override_settings
 
 
 def offline_context_generator():
@@ -942,13 +949,6 @@ class OfflineCompressTestCaseWithLazyStringAlikeUrls(
 
                 self.assertEqual(actual_result, expected_result)
                 self.assertIn(str(settings.COMPRESS_URL), actual_result)
-
-import threading
-import time
-import types
-
-from django.template import Context
-from django.test import TestCase, override_settings
 
 
 @override_settings(COMPRESS_URL="/static/", COMPRESS_URL_PLACEHOLDER="__STATIC__")

